projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_snackbar.scss

```diff
@@ -24,3 +24,16 @@ $_dark-fluent-snackbar: extend($_fluent-snackbar);
 /// @requires {function} extend
 /// @requires $_bootstrap-snackbar
 $_dark-bootstrap-snackbar: extend($_bootstrap-snackbar);
+
+/// Generates a dark indigo snackbar schema.
+/// @type {Map}
```

Comment:
/// @property {map} background [igx-color: ('grays', 400)] - The background color used in the snackbar.